fuzzing/oss-fuzz-scripts/container-environment-bootstrap.sh

```diff
@@ -34,23 +34,24 @@ download_and_concatenate_common_dictionaries() {
   done
 }
 
-fetch_seed_corpra() {
+fetch_seed_corpora() {
   # Seed corpus zip files are hosted in a separate repository to avoid additional bloat in this repo.
   git clone --depth 1 https://github.com/gitpython-developers/qa-assets.git qa-assets &&
     rsync -avc qa-assets/gitpython/corpra/ "$SEED_DATA_DIR/" &&
-    rm -rf qa-assets; # Clean up the cloned repo to keep the Docker image as slim as possible.
+    rm -rf qa-assets # Clean up the cloned repo to keep the Docker image as slim as possible.
 }
 
 ########################
 # Main execution logic #
 ########################
 
-fetch_seed_corpra;
+fetch_seed_corpora
 
 download_and_concatenate_common_dictionaries "$SEED_DATA_DIR/__base.dict" \
   "https://raw.githubusercontent.com/google/fuzzing/master/dictionaries/utf8.dict" \
-  "https://raw.githubusercontent.com/google/fuzzing/master/dictionaries/url.dict";
+  "https://raw.githubusercontent.com/google/fuzzing/master/dictionaries/url.dict"
 
 # The OSS-Fuzz base image has outdated dependencies by default so we upgrade them below.
-python3 -m pip install --upgrade pip;
-python3 -m pip install 'setuptools~=69.0' 'pyinstaller~=6.0'; # Uses the latest versions know to work at the time of this commit.
+python3 -m pip install --upgrade pip
+ # Upgrade to the latest versions known to work at the time the below changes were introduced:
```

Comment:
I suggest removing the leading space here. It occurs to me that this may be intentional, to clarify that the preceding comment continues to apply. But indenting a comment one space may not be noticed, and for anyone who does notice it is indented more than the surrounding code, this may make it look like it applies to the preceding line (since once tends to indent subsequent lines of a command). But this is minor because the presence of the `:` is a more signal to the contrary. So if you prefer to keep this as is, I think that's okay.

Reply:
This was unintentional. I'll remove it. As always, thanks for your attention to detail!

Reply:
Fixed